nikola/__main__.py

```diff
@@ -238,16 +238,13 @@ def __init__(self, nikola, quiet=False):
         self.quiet = quiet
 
     def load_tasks(self, cmd, opt_values, pos_args):
+        DOIT_CONFIG = config.get('DOIT_CONFIG', {})
         if self.quiet:
-            DOIT_CONFIG = {
-                'verbosity': 0,
-                'reporter': 'zero',
-            }
+            DOIT_CONFIG['verbosity'] = 0
+            DOIT_CONFIG['reporter'] = 'zero'
         else:
-            DOIT_CONFIG = {
-                'reporter': ExecutedOnlyReporter,
-                'outfile': sys.stderr,
-            }
+            DOIT_CONFIG['reporter'] = ExecutedOnlyReporter
```

Comment:
updating the dictionary as you suggest makes sense to me.  I hadn't thought that you guys would want anyone to override what you had coded in there.

Since  ``DOIT_CONFIG`` may or may not be in config.   Why don't I just check and pop it out of config in the ``load_tasks`` function?